hbase-client/src/main/java/org/apache/hadoop/hbase/client/MetricsConnection.java

```diff
@@ -118,6 +120,9 @@ static String getScope(Configuration conf, String clusterId, Object connectionOb
 
   private static final String CNT_BASE = "rpcCount_";
   private static final String FAILURE_CNT_BASE = "rpcFailureCount_";
+  private static final String TIMEOUT_CNT_BASE = "rpcExceptionCallTimeout_";
+  private static final String REMOTE_CNT_BASE = "rpcExceptionRemote_";
+  private static final String OTHERS_CNT_BASE = "rpcExceptionOthers_";
```

Comment:
As an operator seeing this metric for the first time I would immediately wonder what "other" means. Can this be defined as something more definitive? "other" can mean anything.